config/webpack/webpack.config.js

```diff
@@ -12,4 +12,6 @@ const envSpecificConfig = () => {
   }
 };
 
-module.exports = envSpecificConfig();
+module.exports = merge(envSpecificConfig(), {
```

Comment:
Incorrect. This doesn't build.

https://github.com/shakacode/react-webpack-rails-tutorial/actions/runs/4327007201/jobs/7555127613

Reply:
My bad.
Fixed now